packages/mask/src/plugins/Wallet/services/chain.ts

```diff
@@ -38,6 +51,13 @@ export async function updateChainState() {
                     ? getBalance(currentAccountSettings.value, {
                           chainId: currentChainIdSettings.value,
                           providerType: currentProviderSettings.value,
+                      }).then((value) => {
```

Comment:
Does the `getBalance` below also need this logic?

Reply:
Here is to update the cache value after each balance is updated